stubs/hdbcli/hdbcli/dbapi.pyi

```diff
@@ -0,0 +1,114 @@
+import decimal
+from _typeshed import ReadableBuffer
+from datetime import date, datetime, time
+from typing import Any, Sequence, Type, overload
+from typing_extensions import Literal
+
+from .resultrow import ResultRow
+
+apilevel: str
+threadsafety: int
+paramstyle: tuple[str, ...]
+connect: Type[Connection]
+
+class Connection:
+    def __init__(
+        self,
+        address: str,
+        port: int,
+        username: str,
+        password: str,
+        autocommit: bool = ...,
+        packetsize: int | None = ...,
+        userkey: str | None = ...,
+        *,
+        sessionvariables: dict[str, str] | None = ...,
+        forcebulkfetch: bool | None = ...,
+    ) -> None: ...
+    def cancel(self) -> bool: ...
+    def close(self) -> None: ...
+    def commit(self) -> None: ...
+    def cursor(self) -> Cursor: ...
+    def getaddress(self) -> str: ...
+    def getautocommit(self) -> bool: ...
+    def getclientinfo(self, key: str = ...) -> str | dict[str, str]: ...
+    def isconnected(self) -> bool: ...
+    def rollback(self) -> None: ...
+    def setautocommit(self, auto: bool = ...) -> None: ...
+    def setclientinfo(self, key: str, value: str | None = ...) -> None: ...
+
+class LOB:
+    def close(self) -> bool: ...
+    def find(self, object: str, length: int, position: int = ...) -> int: ...
+    def read(self, size: int = ..., position: int = ...) -> str | bytes: ...
+    def write(self, object: str | bytes) -> int: ...
+
+_Parameters = Sequence[tuple[Any, ...]]
```

Comment:
```suggestion
_Parameters = Sequence[Tuple[Any, ...]]
```
The current release of mypy doesn't support lowercase `tuple` in this context yet. I think there are a few other instances in this file.

Reply:
This might only affect type aliases, you'll have to check.